docs/operating-eck/installing-eck.asciidoc

```diff
@@ -140,13 +139,14 @@ kubectl delete -n elastic-system \
     clusterrole.rbac.authorization.k8s.io/elastic-operator-view \
     clusterrole.rbac.authorization.k8s.io/elastic-operator-edit \
     clusterrolebinding.rbac.authorization.k8s.io/elastic-operator \
-    rolebinding.rbac.authorization.k8s.io/elastic-operator \
     service/elastic-webhook-server \
+    configmap/elastic-operator \ <1>
     statefulset.apps/elastic-operator \
     validatingwebhookconfiguration.admissionregistration.k8s.io/elastic-webhook.k8s.elastic.co
 ----
+<1> If you have customized the operator configuration you can keep the operator configuration in this ConfigMap and annotate and label it like the CRDs above.
```

Comment:
And so in this case, don't delete it. I'm not sure it's enough clear even it's evident for us.

Isn't it more complex and theorically a user should merge its custom config with the default config provided by the latest version of the operator?

At least, should we have a dedicated bullet point to explain that you have to choose between keeping and annotating the configmap or deleting it?

Reply:
Maybe it is not worth bothering special casing the `ConfigMap`  at all. Maybe we should just turn this into a callout making users aware that deleting the `ConfigMap` will also delete any custom config they have in there?

Reply:
The callout seems a good compromise to me.